crates/ra_hir_def/src/db.rs

```diff
@@ -46,9 +47,12 @@ pub trait DefDatabase: InternDatabase + AstDatabase {
     #[salsa::invoke(RawItems::raw_items_query)]
     fn raw_items(&self, file_id: HirFileId) -> Arc<RawItems>;
 
-    #[salsa::invoke(CrateDefMap::crate_def_map_query)]
+    #[salsa::transparent]
     fn crate_def_map(&self, krate: CrateId) -> Arc<CrateDefMap>;
 
+    #[salsa::invoke(CrateDefMap::compute_crate_def_map)]
```

Comment:
```suggestion
    #[salsa::invoke(CrateDefMap::compute_crate_def_map_query)]
```

Otherwise, LGTM, thanks!

bors d+

Reply:
Done!